### ISSUE TITLE
Allow configuration to be stored in `.azuredevops`

### ISSUE THREAD

**Issue by tomkerkhove** [renovatebot/renovate#10651]
**What would you like Renovate to be able to do?**

Allow configuration to be stored in `.azuredevops`.

**Did you already have any implementation ideas?**

N/A, I don't know how to write Javascript, sorry

**Comment by viceice** [comment]
Please extend your feature request, as it's not really clear what `.azuredevops` should be.

**Comment by tomkerkhove** [comment]
Sorry, with that I meant a folder called like that

**Comment by rarkins** [comment]
@tomkerkhove is this a convention like `.github` is?

**Comment by tomkerkhove** [comment]
Yes, here is another example for PR templates: https://docs.microsoft.com/en-us/azure/devops/repos/git/pull-request-templates?view=azure-devops

**Comment by rarkins** [comment]
I think the requirement is clear, but we need to pause to think about how we can do this manageably. We already have a lot of file names:

<img width="275" alt="image" src="https://user-images.githubusercontent.com/6311784/124085067-f0134b80-da4f-11eb-95a8-a7f664bad303.png">

And with `.yaml` support it will be even more soon. We may want to think how to do this generically instead of expanding the list regularly. BTW this also has an impact on webhook handlers which ideally should react to any config file change, no matter the file name.

**Comment by rarkins** [comment]
Blocked by #10682
